Source/WebCore/platform/graphics/FontDescription.h

```diff
@@ -108,7 +108,7 @@ class FontDescription {
     FontStyleAxis fontStyleAxis() const { return m_fontStyleAxis ? FontStyleAxis::ital : FontStyleAxis::slnt; }
     AllowUserInstalledFonts shouldAllowUserInstalledFonts() const { return static_cast<AllowUserInstalledFonts>(m_shouldAllowUserInstalledFonts); }
     bool shouldDisableLigaturesForSpacing() const { return m_shouldDisableLigaturesForSpacing; }
-    FontPalette fontPalette() const { return m_fontPalette; }
+    const FontPalette& fontPalette() const { return m_fontPalette; }
```

Comment:
FontPalette is essentially an AtomString so this one makes total sense to me, to avoid ref-counting churn.